contracts/upgradeable_contracts/omnibridge_nft/components/common/FailedMessagesProcessor.sol

```diff
@@ -20,9 +20,8 @@ abstract contract FailedMessagesProcessor is BasicAMBMediator, BridgeOperationsS
         require(bridgeContract().failedMessageReceiver(_messageId) == address(this));
         require(bridgeContract().failedMessageSender(_messageId) == mediatorContractOnOtherSide());
 
-        bytes4 methodSelector = this.fixFailedMessage.selector;
-        bytes memory data = abi.encodeWithSelector(methodSelector, _messageId);
-        bridgeContract().requireToPassMessage(mediatorContractOnOtherSide(), data, requestGasLimit());
+        bytes memory data = abi.encodeWithSelector(this.fixFailedMessage.selector, _messageId);
+        _passMessage(data, true);
```

Comment:
Could you share your reasoning why the request to fix the message transfer must go through the PBO lane?

Reply:
Well, I just copied the approach from the original Omnibridge repo, https://github.com/poanetwork/omnibridge/blob/0c581fa83cf762b53912f817100d35a595e0096e/contracts/upgradeable_contracts/components/common/FailedMessagesProcessor.sol#L25 .

Do you think it might be better to always send it to the PBU lane?

Reply:
Ah, I thought you have some ideas why it should go there :) 

Sending to the PBU lane looks more consistent behavior.